Tests/Public/Core/DatabaseValue/DatabaseValueTests.swift

```diff
@@ -109,5 +109,36 @@ class DatabaseValueTests: GRDBTestCase {
         let doubleValue = 1.0.databaseValue
         XCTAssertEqual(intValue, doubleValue)
         XCTAssertEqual(intValue.hashValue, doubleValue.hashValue)
+        
+        let string1 = "foo".databaseValue
+        let string2 = "foo".databaseValue
+        XCTAssertEqual(string1, string2)
+        XCTAssertEqual(string1.hashValue, string2.hashValue)
+        
+        let fooData1 = "foo".dataUsingEncoding(NSUTF8StringEncoding)!.databaseValue
+        let fooData2 = "foo".dataUsingEncoding(NSUTF8StringEncoding)!.databaseValue
+        XCTAssertEqual(fooData1, fooData2)
+        XCTAssertEqual(fooData1.hashValue, fooData2.hashValue)
+    }
+    
+    func testNullDatabaseValueHash() {
+        // Null DatabaseValue has hash 0
+        XCTAssertEqual(DatabaseValue.Null.hashValue, 0)
```

Comment:
If you really really want to raise the code coverage (a disputable goal), please move this test to the Tests/Private folder, the tests than can break without requiring a line in the changelog.